src/services/API/Plio.js

```diff
@@ -27,16 +26,11 @@ export default {
       plioEndpoint += "/play";
     }
 
-    return Promise.all([
-      apiClient().get(plioEndpoint),
-      apiClient().get(itemsEndpoint, {
-        params: { plio: plioId },
-      }),
-    ]).then(([plio, items]) => {
+    return Promise.all([apiClient().get(plioEndpoint)]).then(([plio]) => {
```

Comment:
We don't need `.all()` now right? it was used because we were making multiple calls and wanted to wait for both of them to get resolved.